src/modules/app/actions/listen-to-updates.js

```diff
@@ -180,5 +187,68 @@ export function listenToUpdates() {
         }
       },
     }, err => console.log(err || 'Listening for events'))
+
+    const retryFunc = () => {
+      const retryTimer = 3000
+
+      const retry = (cb) => {
+        const { connection, env, modal } = getState()
+        if (!connection.isConnected) {
+          dispatch(updateModal({
+            ...modal,
+            type: MODAL_NETWORK_DISCONNECTED,
+            canClose: false,
```

Comment:
No need for `canClose` if it’s false.

Reply:
actually since I'm going to use a spread on the modal first, I wanted to explicitly remove close, since another modal might have been currently showing (network mismatch for example). This forces the correct props, don't want canClose to be set to true by accident while trying to reconnect. I'll take another look  and make sure my thinking was correct, but i believe that was why i put that in